tiatoolbox/annotation/storage.py

```diff
@@ -1765,6 +1801,11 @@ def add_from_geojson(
                 at non-baseline resolution.
             origin (Tuple[float, float]):
                 The x and y coordinates to use as the origin for the annotations.
+            import_transform (Callable):
```

Comment:
```suggestion
            transform(Callable):
```

Reply:
Fixed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/721/commits/dd16891f21c00639583e2fdd7817fb0b6c06621f